test/fixtures/functionLayers/serverless.yml

```diff
@@ -0,0 +1,43 @@
+service: service
+provider:
+  name: aws
+
+layers:
+  testLayer:
+    name: default_test_layer
+    path: testLayer
+    description: 'test layer'
+
+  TestLayerWithCapitals:
+    name: capitalized_test_layer
+    path: testLayer
+
+  testLayerWithNoName:
+    path: testLayer
+
+package:
+  exclude:
+    - extra_layers/**
+
+functions:
+  foo:
+    handler: index.handler
+    layers:
+      - { Ref: TestLayerLambdaLayer }
+  bar:
+    handler: index.handler
+  fooBar:
+    handler: index.handler
+    reservedConcurrency: 1
```

Comment:
It'll be nice to give meaningful names (referencing what we test through them) to those lambdas, as we did with below ones